lib/src/flare_animation.dart

```diff
@@ -0,0 +1,213 @@
+part of flame_flare;
+
+/// A class that wraps all the settings of a flare animation from [filename].
+///
+/// It has a similar API to the [FlareActor] widget.
+class FlareActorAnimation {
+  FlareActorAnimation(
+    this.filename, {
+    @required double width,
+    @required double height,
+    this.boundsNode,
+    this.animation,
+    this.fit = BoxFit.contain,
+    this.alignment = Alignment.center,
+    this.isPaused = false,
+    this.snapToEnd = false,
+    this.controller,
+    this.callback,
+    this.color,
+    this.shouldClip = true,
+    this.sizeFromArtboard = false,
+    this.artboard,
+    this.useAntialias = true,
+  }) : flareProvider = null {
+    this.width = width;
+    this.height = height;
+  }
+
+  FlareActorAnimation.asset(
+    this.flareProvider, {
+    @required double width,
```

Comment:
why not use `this.width` and `this.height`?

Reply:
wilco